code/game/jobs/job/cargo_service.dm

```diff
@@ -111,6 +111,7 @@ Bartender
 /datum/outfit/job/bartender
 	name = "Bartender"
 
+	glasses = /obj/item/clothing/glasses/sunglasses/reagent
```

Comment:
I do not think this is going to matter much, especially with the Revs have a majority win rate.